FuadYoussef/TwistOnTime
# Setting up the preset screen

Creating the recycler view for the preset screen. And adding functionality to edit, set, and delete preset timers.

To test:
open preset page -> three timers should appear
Try click the set button to see if the timer appears on the home screen
Try delete timers from the preset page (right now the delete function is only temporary so if you delete a timer, leave the preset screen and then navigate back to the screen all three timers will appear)
Click on the name or duration of a preset timer screen to go to the edit timer screen (note that right now the edit timer screen is not fully set up to handle receiving timers from the preset screen so the back button returns to the home screen and the set button sets a active timer to run. These changes will be addressed by Peyton or Farouk later in the sprint)

There are 3 todos in the code. 2 are for the JSON file and should be done by Fuad the other corresponds to the edit screen and will be done by Peyton or Farouk.
